catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -379,8 +376,7 @@ public Sprite cloneForCloneBrick() {
 	}
 
 	public Sprite cloneForScene() {
-		Sprite clone = clone();
-		return clone;
+		return clone();
```

Comment:
find usages of this functions and change them to call clone() directly and then remove this function